### ISSUE TITLE
Null Fields in Latest Epoch Response

### ISSUE THREAD

**Issue by edridudi** [cardano-community/koios-artifacts#139]
During 99% of Epoch Period,
Trying to get Latest Epoch

`curl --location --request GET 'https://preview.koios.rest/api/v0/epoch_info?limit=1&order=epoch_no.desc'`
returns:
```
[
    {
        "epoch_no": 27,
        "out_sum": null,
        "fees": null,
        "tx_count": null,
        "blk_count": null,
        "start_time": 1668988836,
        "end_time": 1669075236,
        "first_block_time": null,
        "last_block_time": null,
        "active_stake": "360508015510377",
        "total_rewards": null,
        "avg_blk_reward": null
    }
]
```

**Comment by rdlrt** [comment]
As discussed, the state here is replicable when close to epoch transition. Since `active_stake` is desired for some clients (eg: for using leaderlogs), might not be wise to exclude this data, but the change could be to enable showing data for the next epoch by an additional boolean parameter

**Comment by rdlrt** [comment]
Closed via #143
